feat: add support for full page return url (#126)
gr4vy/gr4vy-embed: packages/embed-react/README.md

| `connectionOptions`       | `null`      | An optional set of options passed to a connection when processing a transaction (see https://docs.gr4vy.com/reference#operation/authorize-new-transaction)                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                            |

> | `returnUrl`               | `string`    | An optional return url that that user will be redirected to when embed is being used in an in-app browsers or payments completed in popups where the merchant page has been closed (mobile)                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                           |

Reviewer: ```suggestion
| `returnUrl`               | `string`    | An optional return url that the user will be redirected to when embed is being used in in-app browsers or payments completed in popups where the merchant page has been closed (mobile)                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                           |
```

| `connectionOptions`       | `null`      | An optional set of options passed to a connection when processing a transaction (see https://docs.gr4vy.com/reference#operation/authorize-new-transaction)                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                            |

> | `fullPageReturnUrl`       | `string`    | An optional return url that the user will be redirected to when embed is being used in in-app browsers or payments completed in popups where the merchant page has been closed (mobile)                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               |

Reviewer 2: ```suggestion
| `fullPageReturnUrl`       | `string`    | An optional return url that the user will be redirected to when Embed is being used in in-app browsers, or when payments are completed in popups where the merchant page has been closed (mobile)                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               |
```

> | `redirectMode`            | `fallback`  | `fallback`, `fullpage` - An optional configuration to change how redirects to payment providers are performed. Auto will attempt a popup before redirecting, `redirect` will always redirect. This can be useful for testing purposes.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                |

Reviewer 2: This still mentions `auto`.

> | `redirectMode`            | `fallback`  | `fallback`, `fullPage` - An optional configuration to change how redirects to payment providers are performed. `fallback` will attempt a popup before redirecting, `fullPage` will always redirect. This can be useful for testing purposes.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          |

Reviewer 2: ```suggestion
| `redirectMode`            | `fallback`  | `fallback`, `fullPage` - An optional configuration to change how redirects to payment providers are performed. `fallback` will attempt a popup before redirecting, `fullPage` will always perform a full page redirect rather than using a popup. This can be useful for testing the fallback behaviour where a popup could not be used.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          |
```
